### ISSUE TITLE
Update PokemonGo-Map

### ISSUE THREAD

**Issue by noktur** [engle2192/PoGoMap-GUI#48]
Since the new API is out and PokemonGo-Map has already updated, it should be updated to use the latest version here too.

**Comment by Go2Engle** [comment]
This is true, but the account creator no longer works as the one I had implemented does not work anymore. Also I just dont have any motivation to even download and update my repo with the latest PokemonGo-Map.
